sql/core/src/test/resources/tpcds-plan-stability/approved-plans-v1_4/q12.sf100/explain.txt

```diff
@@ -86,15 +86,15 @@ Input [9]: [ws_item_sk#1, ws_ext_sales_price#2, ws_sold_date_sk#3, i_item_sk#6,
 Output [2]: [d_date_sk#13, d_date#14]
 Batched: true
 Location [not included in comparison]/{warehouse_dir}/date_dim]
-PushedFilters: [IsNotNull(d_date), GreaterThanOrEqual(d_date,1999-02-22), LessThanOrEqual(d_date,1999-03-24), IsNotNull(d_date_sk)]
+PushedFilters: [IsNotNull(d_date), GreaterThanOrEqual(d_date,1999-02-22), IsNotNull(d_date_sk)]
```

Comment:
Now that `INTERVAL 30 days` is `DayTimeIntervalType` and the result of `cast('1999-02-22' AS DATE) + INTERVAL 30 days` is `TimestampType`so filter push down seems not to work.